README.md
# biz.jmaconsulting.oapproviderlistapp

CHANGELOG.md

 #### 2019-11-30 01:11:20 +0530 [minor change in script](https://github.com/JMAConsulting/biz.jmaconsulting.oapproviderlistapp/commit/ba4ee13) <br> Monish Deb (committed by Monish Deb)<br><br>

 #### 2019-11-30 01:10:18 +0530 [minor change in script](https://github.com/JMAConsulting/biz.jmaconsulting.oapproviderlistapp/commit/304fb98) <br> Monish Deb (committed by Monish Deb)<br><br>

 #### 2019-11-30 01:06:55 +0530 [add post-commit script](https://github.com/JMAConsulting/biz.jmaconsulting.oapproviderlistapp/commit/7c0e878) <br> Monish Deb (committed by Monish Deb)<br><br>

 #### 2019-11-30 00:12:31 +0530 [minor changes](https://github.com/JMAConsulting/biz.jmaconsulting.oapproviderlistapp/commit/24d24b2) <br> Monish Deb (committed by Monish Deb)<br><br>

 #### 2019-11-30 00:11:21 +0530 [CH-54 : Add shellscript to create script to generate changelog](https://github.com/JMAConsulting/biz.jmaconsulting.oapproviderlistapp/commit/b66a3d2) <br> Monish Deb (committed by Monish Deb)<br><br>

 #### 2019-11-29 08:58:24 +0530 [remove test comment](https://github.com/JMAConsulting/biz.jmaconsulting.oapproviderlistapp/commit/5e471d5) <br> Monish Deb (committed by Monish Deb)<br><br>

 #### 2019-11-29 08:57:21 +0530 [minor change to add test comment](https://github.com/JMAConsulting/biz.jmaconsulting.oapproviderlistapp/commit/730ea93) <br> Monish Deb (committed by Monish Deb)<br><br>

 #### 2019-11-29 08:50:53 +0530 [remove comment](https://github.com/JMAConsulting/biz.jmaconsulting.oapproviderlistapp/commit/2256cb5) <br> Monish Deb (committed by Monish Deb)<br><br>

 #### 2019-11-29 08:46:37 +0530 [Add change log changes](https://github.com/JMAConsulting/biz.jmaconsulting.oapproviderlistapp/commit/99c260f) <br> Monish Deb (committed by Monish Deb)<br><br>

 #### 2019-11-29 08:25:28 +0530 [test change log changes](https://github.com/JMAConsulting/biz.jmaconsulting.oapproviderlistapp/commit/0e2e16f) <br> Monish Deb (committed by Monish Deb)<br><br>

 #### 2019-11-28 20:41:09 +0530 [OAPS-119 fix](https://github.com/JMAConsulting/biz.jmaconsulting.oapproviderlistapp/commit/6961055) <br> Monish Deb (committed by Monish Deb)<br><br>

 #### 2019-11-21 19:15:17 +0530 [OAPS-147 fix](https://github.com/JMAConsulting/biz.jmaconsulting.oapproviderlistapp/commit/e100967) <br> Monish Deb (committed by Monish Deb)<br><br>

 #### 2019-11-21 19:14:13 +0530 [OAPS-147 fix](https://github.com/JMAConsulting/biz.jmaconsulting.oapproviderlistapp/commit/8f4952c) <br> Monish Deb (committed by Monish Deb)<br><br>

 #### 2019-11-12 14:43:40 +0100 [Added link colors](https://github.com/JMAConsulting/biz.jmaconsulting.oapproviderlistapp/commit/ff83b95) <br> Edsel (committed by Edsel)<br><br>

 #### 2019-11-11 12:50:16 +0100 [Bug fixes](https://github.com/JMAConsulting/biz.jmaconsulting.oapproviderlistapp/commit/6b281d8) <br> Edsel (committed by Edsel)<br><br>

 #### 2019-11-11 17:14:11 +0530 [Multiple additions to listings](https://github.com/JMAConsulting/biz.jmaconsulting.oapproviderlistapp/commit/f341978) <br> Edsel Lopez (committed by Edsel Lopez)<br><br>

 #### 2019-11-06 16:25:06 +0530 [minor fix](https://github.com/JMAConsulting/biz.jmaconsulting.oapproviderlistapp/commit/edcb649) <br> Monish Deb (committed by Monish Deb)<br><br>

 #### 2019-11-06 16:23:15 +0530 [minor fix](https://github.com/JMAConsulting/biz.jmaconsulting.oapproviderlistapp/commit/7945066) <br> Monish Deb (committed by Monish Deb)<br><br>

 #### 2019-10-24 18:50:57 +0530 [minor fix](https://github.com/JMAConsulting/biz.jmaconsulting.oapproviderlistapp/commit/da0c146) <br> Monish Deb (committed by Monish Deb)<br><br>

 #### 2019-10-24 18:31:17 +0530 [minor fix](https://github.com/JMAConsulting/biz.jmaconsulting.oapproviderlistapp/commit/5fd90da) <br> Monish Deb (committed by Monish Deb)<br><br>

 #### 2019-10-24 14:51:39 +0530 [add approval date in report](https://github.com/JMAConsulting/biz.jmaconsulting.oapproviderlistapp/commit/0fc2c01) <br> Monish Deb (committed by Monish Deb)<br><br>

 #### 2019-10-24 14:26:44 +0530 [add approval date in report](https://github.com/JMAConsulting/biz.jmaconsulting.oapproviderlistapp/commit/096bb59) <br> Monish Deb (committed by Monish Deb)<br><br>

 #### 2019-10-14 19:55:20 +0530 [OAPS-131 fix](https://github.com/JMAConsulting/biz.jmaconsulting.oapproviderlistapp/commit/a426662) <br> Monish Deb (committed by Monish Deb)<br><br>

 #### 2019-10-14 19:52:34 +0530 [OAPS-131 fix](https://github.com/JMAConsulting/biz.jmaconsulting.oapproviderlistapp/commit/c184960) <br> Monish Deb (committed by Monish Deb)<br><br>

 #### 2019-10-14 19:48:25 +0530 [OAPS-131 fix](https://github.com/JMAConsulting/biz.jmaconsulting.oapproviderlistapp/commit/cd92cfa) <br> Monish Deb (committed by Monish Deb)<br><br>

 #### 2019-09-24 00:18:49 +0530 [OAPS-36 fix](https://github.com/JMAConsulting/biz.jmaconsulting.oapproviderlistapp/commit/20faf54) <br> Monish Deb (committed by Monish Deb)<br><br>

 #### 2019-09-24 00:16:06 +0530 [OAPS-36 fix](https://github.com/JMAConsulting/biz.jmaconsulting.oapproviderlistapp/commit/5e67f99) <br> Monish Deb (committed by Monish Deb)<br><br>

 #### 2019-09-24 00:09:52 +0530 [OAPS-36 fix](https://github.com/JMAConsulting/biz.jmaconsulting.oapproviderlistapp/commit/8206797) <br> Monish Deb (committed by Monish Deb)<br><br>

 #### 2019-09-23 23:48:08 +0530 [OAPS-125 fix](https://github.com/JMAConsulting/biz.jmaconsulting.oapproviderlistapp/commit/96b178b) <br> Monish Deb (committed by Monish Deb)<br><br>

 #### 2019-09-23 22:50:38 +0530 [OAPS-126 fix](https://github.com/JMAConsulting/biz.jmaconsulting.oapproviderlistapp/commit/be5c944) <br> Monish Deb (committed by Monish Deb)<br><br>

 #### 2019-09-23 22:43:13 +0530 [OAPS-126 fix](https://github.com/JMAConsulting/biz.jmaconsulting.oapproviderlistapp/commit/527fb6c) <br> Monish Deb (committed by Monish Deb)<br><br>


oapproviderlistapp.variables.php
<?php

define('OAPPROVIDERLIST', 14);

// Education Custom Group
define('EDUCATION', 'custom_28');
define('DEGREE', 'custom_29');
define('YEAR', 'custom_30');

// Employment History
define('ORG', 'custom_32');
define('TITLE', 'custom_33');
define('DATES', 'custom_34');
define('TASKS', 'custom_35');
define('TOTAL_HOURS', 'custom_36');
define('SUPER_HOURS', 'custom_37');
define('SUPER_CONTACT', 'custom_38');

// Signature
define('SIGNATURE', 'custom_39');

define('OAP_INDIVIDUAL', 16);
define('OAP_PHONEADDRESS', 17);
define('OAP_PROFESSIONAL', 18);
define('OAP_EXPERIENCE', 19);
define('OAP_SECTORCHECK', 20);
define('OAP_INSURANCE', 21);
define('OAP_DOCUMENTATION', 22);
define('OAP_SIGNATURE', 23);
define('OAP_OTHER_EMP', 'civicrm_value_other_employe_16');
define('OAP_OTHER_PRO', 'civicrm_value_other_profess_12');
define('OAP_EMP_HIS', 'civicrm_value_employment_hi_10');
define('OAP_LISTING', 28);
